.circleci/config.yml

```
@@ -0,0 +1,199 @@
+version: 2.1
+orbs:
+  aws-s3: circleci/aws-s3@1.0.9
+
+environment:
+  DEBUG: true
+
+# -------------------------------------------------------------------------
+# jobs definition ---------------------------------------------------------
+# -------------------------------------------------------------------------
+set_default_excuter: &set_default_excuter
+  docker:
+    - image: node:carbon-alpine
+  working_directory: ~/select-frontend
+
+restore_saved_cache: &restore_saved_cache
+  restore_cache:
+    key: select-frontend-{{ checksum "package.json" }}
+
+save_build_cache: &save_build_cache
+  save_cache:
+    key: select-frontend-{{ checksum "package.json" }}
```

`node_modules`의 상태는 `yarn.lock`이 더 잘 표현하기 때문에 캐시 키로 `checksum "yarn.lock"`을 쓰는 게 좋을 것 같습니다. `package.json`에는 디펜던시 말고도 많으니까요.

```suggestion
    key: select-frontend-{{ checksum "yarn.lock" }}
```